sql/moz-fx-data-shared-prod/firefox_ios_derived/retention_v1/metadata.yaml

```diff
@@ -0,0 +1,37 @@
+friendly_name: Client Retention - Firefox iOS (Aggregated)
+description: |
+  Client Retention (Firefox iOS) metrics aggregated on:
+  - metric_date
+  - first_seen_date
+  - app_name
+  - channel
+  - country
+  - adjust_ad_group
+  - adjust_campaign
+  - adjust_creative
+  - adjust_network
+
+  For all clients that sent us a ping on the metric date.
+  Client's attributes are the client's attributes observed on the metric date.
+
+  see: DENG-3183 for more information.
+owners:
+- kik@mozilla.com
+labels:
+  schedule: daily
+  incremental: true
+scheduling:
+  dag_name: bqetl_firefox_ios
+  depends_on_past: false
+  date_partition_parameter: submission_date
+bigquery:
+  time_partitioning:
+    type: day
+    field: metric_date
+    require_partition_filter: false
+  clustering:
+    fields:
```

Comment:
change to app_name, country, first_seen_date

I'm not sure about channel. I see the benefit to test on smaller queries, but hopefully this aggregated table won't be too big and therefore we won't need it. Almost everything is release anyway.